app/commands/ingest.py

```
@@ -100,95 +100,99 @@ def run():
             active = "active" if ch.get("is_active") else "inactive"
+            config = ch.get("config", {})
             logger.info(
-                f"  [{active}] {ch['channel_name']} "
-                f"(priority: {ch.get('priority', '-')}, "
-                f"category: {ch.get('category', '-')}, "
+                f"  [{active}] {ch['name']} "
+                f"(priority: {config.get('priority', '-')}, "
+                f"category: {config.get('category', '-')}, "
                 f"id: {ch['id']})"
             )
     except Exception as e:
```

Ruff (0.15.13)

[warning] Do not catch blind exception: `Exception`

(BLE001)